### PR TITLE
[music3] disable lighting for meshes in 2D

### PR DESCRIPTION
**Pull request by mathildemerle**
From PR https://github.com/Inria-Asclepios/medInria-public/pull/514 and https://github.com/Inria-Asclepios/medInria-public/pull/351
Commit https://github.com/medInria/medInria-public/commit/d716ccdbea0f85c24e419bdaffd88489bc038885 and https://github.com/medInria/medInria-public/pull/396/commits/632b154611caa50d01d85f1c3c557f8ea93f2b86

>     * Fix lighting problem that caused mesh contours to be barely visible
> 
>     * Increase line width from 1 to 3 pixels.
> 
> 
> Before:
> b4
> 
> After:
> afta

### CODE REVIEW

**Comment by mathildemerle** [comment]
I asked @mjuhoor your question @ocommowi @Florent2305 and the colors remains on the 2D mesh :)

**Comment by Florent2305** [comment]
great

File: src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.cxx [hunk 1]
```diff
@@ -69,12 +83,17 @@ void vtkMetaDataSetSequence::Initialize()
...
+    return new vtkMetaDataSetSequence(*this);
+}
+
 //----------------------------------------------------------------------------
 void vtkMetaDataSetSequence::AddMetaDataSet (vtkMetaDataSet *metadataset)
 {
     if (!metadataset)
     {
-        vtkErrorMacro(<<"NULL object !"<<endl);
+        vtkErrorMacro(<<"nullptr object !"<<endl);
```
Comment (Florent2305) [inline src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.cxx]: ;) query replace

File: src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.h [hunk 2]
```diff
@@ -228,10 +231,7 @@ class MEDVTKDATAMESHBASE_EXPORT vtkMetaDataSetSequence: public vtkMetaDataSet
   std::vector<vtkMetaDataSet*> MetaDataSetList;
   //ETX
 
-  
  private:
-  
-  vtkMetaDataSetSequence(const vtkMetaDataSetSequence&);      // Not implemented.
   void operator=(const vtkMetaDataSetSequence&);              // Not implemented.
```
Comment (Florent2305) [inline src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.h]: It's not matters but, with modern c++ to put constructor/destructor/operator not accessible it's:
 void operator=(const vtkMetaDataSetSequence&) = delete; directly into public section.

File: src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSet.h [hunk 3]
```diff
@@ -5,32 +5,42 @@
...
 
 #include <vtkDataObject.h>
+#include <vtkDataSet.h>
+
 #include <string>
 #include <vector>
 
-#include <vtkDataSet.h>
+#ifdef WIN32
+    #ifndef NAN
```
Comment (Florent2305) [inline src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSet.h]: I dont like that

**Pull request merged**